docs/topics/Permissions.md

```diff
@@ -16,7 +18,7 @@ Permissions in Discord are stored within a 53-bit integer and are calculated usi
 | MANAGE\_GUILD * | `0x00000020` | Allows management and editing of the guild |
 | ADD\_REACTIONS | `0x00000040` | Allows for the addition of reactions to messages |
 | VIEW\_AUDIT\_LOG | `0x00000080` | Allows for viewing of audit logs |
-| READ\_MESSAGES | `0x00000400` | Allows reading messages in a channel. The channel will not appear for users without this permission |
+| VIEW\_CHANNEL | `0x00000400` | Allows guild members to view a channel, which includes reading messages in text channels |
```

Comment:
Description can be `Allows guild members to read text channels and view voice channels`.

Reply:
Here, I am trying to make the distinction between viewing the channel in the channel list (which applies to text, voice, and category) and reading messages. "Allows guild members to read text channels and view voice channels" ignores the visibility of text and category channels in the list.

Reply:
This is OK to me. I actually think the in-app text should be updated, so I'll bump someone on that.

Reply:
@SinisterRectus Hokay I'm good with this too then.